examples/grpc-bridge/Dockerfile-grpc

```diff
@@ -1,7 +1,7 @@
 FROM lyft/envoy:latest
 
 RUN mkdir /var/log/envoy/
-COPY ./bin/service /usr/local/bin/srv
```

Comment:
At the end of the example there are instructions for building the go service which creates a bin/service binary. Is this COPY line supposed to copy the result of that? If so the order of the tutorial should be changed.

https://lyft.github.io/envoy/docs/install/sandboxes.html#building-the-go-service